src/libServer/EthRpcMethods.cpp

```diff
@@ -739,6 +744,146 @@ std::string EthRpcMethods::GetEthEstimateGas(const Json::Value& json) {
   }
 }
 
+std::string EthRpcMethods::NewGetEthEstimateGas(const Json::Value& json) {
```

Comment:
I think you don't have to rename the method, you can replace it. 
(And `New...` is not a good name - one day it will be old enough)

Reply:
Its like that for this check in as it passes all tests with the old routine, this is the last routine we are now working on.

Hopefully very soon, the version with New at the front will replace the old one.